InnoViber/Middleware/ExeptionHandlerMiddleware.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace InnoViber.API.Middleware;
+
+public class ExeptionHandlerMiddleware
+{
+    private readonly RequestDelegate _next;
+
+    private readonly ILogger<ExeptionHandlerMiddleware> _logger;
+
+    public ExeptionHandlerMiddleware(RequestDelegate next, ILogger<ExeptionHandlerMiddleware> logger)
+    {
+        _next = next;
+        _logger = logger;
+    }
+
+    public async Task Invoke(HttpContext httpContext)
```

Comment:
U can also customize status code and send status code depends on exception or for ex always send 500 to hide some BE error details from user